packages/paste-core/components/badge/src/styles.ts

```diff
@@ -78,6 +78,21 @@ export const badgeVariantStyles: {
     color: "colorTextDecorative40",
     boxShadow: "shadowBorderDecorative40Weaker",
   },
+  brand10: {
```

Comment:
Another thought: 

![image](https://github.com/twilio-labs/paste/assets/1592327/7c82d3ec-0750-4505-8132-15ee34017fc1)
This may be more clear as `brandConsole/Segment/SendGrid` so people don't just use them as new color options, but instead as semantically necessary to illustrate a brand color.  It would be unideal if they were used a brand colors and Console made heavy use of Segment's brand color.

Reply:
cc @serifluous

Reply:
The brand team wants to move away from having brand colors map to specific products, which makes sense since Email is going to be moved into Console soon. The brand10/20/30 naming makes it a little more flexible to whatever happens when the products are more unified.

See this example on twilio.com. Blue is used for Flex:
<img width="1176" alt="Screenshot 2024-03-06 at 9 25 37 AM" src="https://github.com/twilio-labs/paste/assets/3821840/35474392-130e-4253-89d9-bbac057b63a3">